src/index.js

```diff
@@ -207,6 +208,13 @@ export async function showLocation(options) {
       url = `${prefixes['gett']}order?pickup=my_location&dropoff_latitude=${lat}&dropoff_longitude=${lng}`
 
       break
+    case 'navermap':
+      checkNaverMapOptions(options)
```

Comment:
Change above makes this extra check obsolete. I think it's better to just not show the 'Naver' option to users if the required option isn't set, to prevent unexpected errors/crashes.

Reply:
that's great! 100% agreed. I've applied this change, should I squash that new commit into this one before push?

Reply:
Great! Separate commits are fine.